README.md

```diff
@@ -205,13 +205,16 @@ Darkflame Universe utilizes a MySQL/MariaDB database for account and character i
 
 Initial setup can vary drastically based on which operating system or distribution you are running; there are instructions out there for most setups, follow those and come back here when you have a database up and running.
 * Create a database for Darkflame Universe to use
-* Use the command `./MasterServer -m` to automatically run them.
 
-**Configuration**
+#### Configuration
 
 After the server has been built there should be four `ini` files in the build director: `authconfig.ini`, `chatconfig.ini`, `masterconfig.ini`, and `worldconfig.ini`. Go through them and fill in the database credentials and configure other settings if necessary.
 
-**Verify**
+#### Initialization and Migrations
+
+Use the command `./MasterServer -m` to automatically initialize the database and run any migrations.
```

Comment:
This command does not initialize the database.  We should not be doing the database initialization work and should require our users to create that on their own merit.  I suggest changing the phrasing here to `... to setup the tables in the database.  The first time this command is run on a database, the tables will be up to date with the most recent version.  To update your database tables, run this command again.  Multiple invocations will not affect any functionality.`

Reply:
Updated wording